packages/@okta/vuepress-site/guides/sign-into-mobile-app/before-you-begin/iOS/create-app.md

```diff
@@ -0,0 +1,3 @@
+The [iOS quickstart](android link here) teaches you the basics of building an Android application.
```

Comment:
Well that's confusing...

Reply:
Ha, yeah - this is just a placeholder for the iOS quickstart...I'll update the text.